mopro-core/build.rs

```diff
@@ -79,8 +79,10 @@ fn build_dylib(wasm_path: String, dylib_name: String) -> Result<()> {
 
 fn main() -> Result<()> {
```

Comment:
This is very messy... I'm working on making this easier to work with. Will update once fixed, then we can make this easier to merge

Reply:
Ok let me know, how I should resolve once done

Reply:
Fixed: https://github.com/oskarth/mopro/pull/67/files#diff-10095d80195138814b66f1d6f5b769b0763cd537d9ef563a28c6d75444a151d5

See https://github.com/oskarth/mopro/pull/67/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R34 for what to do

I suggest adding an `anon-aadhaar-config.toml` to this PR